README.md

```diff
@@ -77,8 +77,8 @@ var adapter = metamorphoses.apiBlueprintAdapter;
 // or the legacy Blueprint Parser (https://github.com/apiaryio/blueprint-parser)
 parse(source, ..., function (err, result) {
 
-  // transform the error object
-  var err = adapter.transformError(source, err);
+  // transform the error parse result into an error
```

Comment:
I'm not sure, it's not retrieving it, since it is returning a different error derived from the error in the parse result in some cases.